frontend/src/pages/Home/Home.tsx

```diff
@@ -25,15 +29,17 @@ const HomePageRecentItems = () => {
                 size="small"
                 dataSource={recents}
                 renderItem={(item, index) => (
-                    <List.Item>
-                        <List.Item.Meta title={item.name} />
-                        <Button
+                    <ItemListItem 
+                        item={item} 
+                        path={`/items/${item.token}`} 
+                        actions = {[<Button
                             type="primary"
-                            href={`/activities/${item.item_type}/${item.token}`}
+                            onClick={e => {
```

Comment:
in general, we should prefer wrapping <Button> in <Link> / setting an href to having  navigate in onClick - gives browsers more information